OJP/OJP_Lines.xsd

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="http://www.vdv.de/ojp" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:siri="http://www.siri.org.uk/siri" targetNamespace="http://www.vdv.de/ojp" elementFormDefault="qualified" attributeFormDefault="unqualified" id="OJP_siri_service">
+	<xs:annotation>
+		<xs:documentation>OJP/OJP_Lines.xsd - OJP Line information request and response</xs:documentation>
+	</xs:annotation>
+	<xs:import namespace="http://www.siri.org.uk/siri" schemaLocation="../siri/siri_model/siri_all-v2.0.xsd"/>
+	<xs:include schemaLocation="OJP_All.xsd"/>
+	<xs:import namespace="http://www.siri.org.uk/siri" schemaLocation="../siri/siri_model/siri_modes-v1.1.xsd"/>
+	<xs:annotation>
+		<xs:documentation>=============================== Line Information Service ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:annotation>
+		<xs:documentation>=============================== Line Information Response Group ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:group name="LineInformationResponseGroup">
+		<xs:annotation>
+			<xs:documentation>Line Information response group.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="LineResult" type="LineResultStructure" minOccurs="0" maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:group>
+	<xs:annotation>
+		<xs:documentation>=============================== LineInformation support functions ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:complexType name="LineResultStructure">
+		<xs:sequence>
+			<xs:element name="LineRef" type="siri:LineRefStructure">
+				<xs:annotation>
+					<xs:documentation>Reference to the LINE.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element ref="siri:PublishedLineName" minOccurs="0"/>
+			<xs:element name="DirectionRef" type="siri:DirectionRefStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>DIRECTION of LINE if relevant for the shape.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Shape" type="siri:LineShapeStructure" minOccurs="0">
```

Comment:
@Aurige : Could you answer?

Reply:
Aurige: Would prefer NeTEx and use of gml. So here only GML. GML is very heavy payload, because of that. Google has a a lighter polyline code. Proposal using GML will be made.